frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -29,7 +29,7 @@ table.table.table-in-card.c-payments(:class='{"is-editing": paymentsType === "ed
 
         // TODO: replace condition to indicate whether or not the payment date is < or > than the current date using payment.paymentStatusText
         i18n.c-user-month(
-          :class='index === 0 ? "has-text-1" : "pill is-danger is-small"'
+          :class='index === 0 ? "has-text-1" : "pill is-danger"'
```

Comment:
Are the changes in this file (e.g. removal of `is-small`) because the previous version didn't follow the designs?

Reply:
`is-small` isn't needed. It doesn't exist a "small" variant of the `pill` component. They are all the same size.